presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py

```diff
@@ -1 +1,13 @@
-"""Main module."""
+from flask import Flask, request
```

Comment:
We should decide if the flask app is part of the package (i.e. inside the `presidio_anonymizer` folder) or outside, in some `app.py` file, which is similar to how we had the grpc server set up.

Reply:
Not sure I understand the remark. 
The entire application is going to be inside the presidio_anonymizer folder just like in the anonymizer (I am trying to keep consistency between the applications).
If I am not mistaking @itye-msft and I are per-programming the flask on the analyzer, so if the layout will change, we will also change the anonymizer.